frontend/src/components/c-summary-charts.vue

```diff
@@ -875,6 +913,38 @@ export default defineComponent({
 
       return totalContribution / totalCommits;
     },
+
+    setHighlighted(groupIndex: number, index: number): void {
+      this.highlightedRepo = groupIndex;
+      this.highlightedUser = index;
+
+      window.addHash('highlightedRepo', groupIndex.toString());
+      window.addHash('highlightedUser', index.toString());
+      window.encodeHash();
+
+      this.highlightRepo(groupIndex, index);
+    },
+
+    highlightRepo(groupIndex: number, index: number): void {
+      const offset = this.filteredRepos
```

Comment:
This part is a bit strange because of the way the refs are stored for summary-chart. Basically, there are multiple groups and in each group, multiple repos. But the number X in the ref for `summary-chart-X` refers to the index of the repo in the individual group. This means that in each ref, there is actually an array of div elements from multiple groups. I didn't want to change the way the refs were stored since the iframe feature seems to depend on it